service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceLocators.scala

```diff
@@ -73,14 +80,34 @@ trait CircuitBreakerComponents {
   def circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider
 
   lazy val circuitBreakerConfig: CircuitBreakerConfig = new CircuitBreakerConfig(configuration)
-  lazy val circuitBreakers = new CircuitBreakers(actorSystem, circuitBreakerConfig, circuitBreakerMetricsProvider)
+
+  // for backward compatibility we still need to provide it for wiring
+  lazy val circuitBreakers: CircuitBreakers13 = {
+
+    // only in case some 3rd part lib are still wiring the old one.
+    println( // (no logger available here, so good old println)
```

Comment:
Same case as above, we need to make it available for wiring also on the Scala side. 

This warning is expected to be print out if the old `CircuitBreakers` get inadvertently wired.